### ISSUE TITLE
Remote Development => utils - ERROR - An SSL Error occurred

### ISSUE THREAD

**Issue by caadecarvalho** [facebook/nuclide#1305]
### Issue and Steps to Reproduce
When I try to start add a Remote Project Folder, it returns this error: 

Unexpected error occurred: Traceback (most recent call last): File "scripts/nuclide_server_manager.py", line 406, in json.dump(result, f) File "/usr/lib64/python2.7/json/init.py", line 189, in dump for chunk in iterable: File "/usr/lib64/python2.7/json/encoder.py", line 434, in _iterencode for chunk in _iterencode_dict(o, _current_indent_level): File "/usr/lib64/python2.7/json/encoder.py", line 408, in _iterencode_dict for chunk in chunks: File "/usr/lib64/python2.7/json/encoder.py", line 332, in _iterencode_list for chunk in chunks: File "/usr/lib64/python2.7/json/encoder.py", line 442, in _iterencode o = _default(o) File "/usr/lib64/python2.7/json/encoder.py", line 184, in default raise TypeError(repr(o) + " is not JSON serializable") TypeError: is not JSON serializable .
Original error message:
 Traceback (most recent call last):
  File "scripts/nuclide_server_manager.py", line 406, in <module>
    json.dump(result, f)
  File "/usr/lib64/python2.7/json/__init__.py", line 189, in dump
    for chunk in iterable:
  File "/usr/lib64/python2.7/json/encoder.py", line 434, in _iterencode
    for chunk in _iterencode_dict(o, _current_indent_level):
  File "/usr/lib64/python2.7/json/encoder.py", line 408, in _iterencode_dict
    for chunk in chunks:
  File "/usr/lib64/python2.7/json/encoder.py", line 332, in _iterencode_list
    for chunk in chunks:
  File "/usr/lib64/python2.7/json/encoder.py", line 442, in _iterencode
    o = _default(o)
  File "/usr/lib64/python2.7/json/encoder.py", line 184, in default
    raise TypeError(repr(o) + " is not JSON serializable")
TypeError: <logging.LogRecord object at 0x105fc90> is not JSON serializable

### Expected Behavior
Starts and works remotely over nuclide server.

### Actual Behavior
It starts nuclide server remotely but the log file is full of SSL issue:

nuclide.log:
2017-09-18 18:13:21.541 INFO (pid:5230) nuclide-server - NuclideServer started on port 9092.
2017-09-18 18:13:21.541 INFO (pid:5230) nuclide-server - Using node v8.4.0.
2017-09-18 18:13:21.541 INFO (pid:5230) nuclide-server - Server ready time: 605ms

server-start/nuclide.log:
.
.
.
2017-09-18 18:13:23,661 - utils - ERROR - An SSL Error occurred
2017-09-18 18:13:23,661 - utils - ERROR - An SSL Error occurred
2017-09-18 18:13:23,661 - utils - ERROR - An SSL Error occurred
.
.
.

### Versions
* Atom: 1.20.0 x64
* Nuclide: 0.254.0
* Client OS: MacOS Sierra
* Server OS (optional): CentOS Linux release 7.3.1611 (Core) 3.10.0-514.10.2.el7.x86_64

**Comment by rmatev** [comment]
I hit the same problem. The root issue is described at https://access.redhat.com/articles/2039753#controlling-certificate-verification-7

tl;dr: workaround by e.g. adding `export PYTHONHTTPSVERIFY=0` before the python invocation in `nuclide-start-server`

**Comment by caadecarvalho** [comment]
`export PYTHONHTTPSVERIFY=0 && nuclide-start-server`

You made it, I own you a beer! =]
